From 35c738eb58c990a2b46c117aeae0e1685daea1a7 Mon Sep 17 00:00:00 2001
From: Ming Tan <ming.tan@intel.com>
Date: Mon, 21 Jan 2019 11:06:34 +0800
Subject: [PATCH 17/18] set_storage: Fix the sepolicy issue when set API level
 to 28.

After set API level to 28, it will report deny some operation for
default_prop:file.

Tracked-On: OAM-75347
Signed-off-by: Ming Tan <ming.tan@intel.com>
---
 sepolicy/set_storage/set_storage.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sepolicy/set_storage/set_storage.te b/sepolicy/set_storage/set_storage.te
index dbbff4a..ef730e0 100644
--- a/sepolicy/set_storage/set_storage.te
+++ b/sepolicy/set_storage/set_storage.te
@@ -29,3 +29,4 @@ allow set_storage block_device:blk_file { getattr setattr };
 # allow set_storage frp_block_device:blk_file setattr;
 
 allow set_storage set_storage:capability { fowner chown };
+allow set_storage default_prop:file { getattr map open read setattr };
-- 
1.9.1

